app/models/customer.rb

```diff
@@ -1,11 +1,16 @@
 class Customer < ApplicationRecord
   has_many :invoices
+  has_many :transactions, through: :invoices
```

Comment:
Brilliant!

Reply:
I honestly didn't know this was possible. I remember you asking early on if this was a thing we could do, super cool to see that it is :D Well done!